### ISSUE TITLE
Feature suggestion: Make report-unused-disable-directives report unknown rules

### ISSUE THREAD

**Issue by jrencz** [eslint/eslint#9647]
**Tell us about your environment**

* **ESLint Version:** 4.11
* **Node Version:** 9.2
* **npm Version:** 5.5.1

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:** n/a

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

Long time ago I introduced decorators to my project. It turned out I can't put decorator just above a class if I have `require-jsdoc` eto to `error`. I tried putting decorator above the doc block, but then `lines-around-comment` complained. So I figured out in such cases I want it to be quiet. **But I made a typo**. I added `s` at the end of the rule name. **But it did help** even though not because of what was in the inline comment, rather because the line comment is just there.

Then I copy-pasted this into many more files.
I end up with something like this:

```js
@someDecorator()
// eslint-disable-next-line lines-around-comments
/**
 * ...
 */
class Foo {}
```

Recently i tried `--report-unused-disable-directives`

```bash
eslint --report-unused-disable-directives someFile.js
```

**What did you expect to happen?**
I'd expect for ESLint to tell me that there's no rule `lines-around-comments` (ending with `s`) in my config. 
It would be super handy (but not required) if it told me there's `lines-around-comment` and I may have made a typo.

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint told me: `Unused eslint-disable directive (no problems were reported from 'lines-around-comments')` which is technically correct but it's misleading to a certain extent.

I suggest `--report-unused-disable-directives` flag should first report attempts to silence out rules which doesn't exist in the config (for whatever reason: whether it's a typo or the rule was in the config one day and it's not there any more)

I'd like to than @didnotwant for helping me figure out what was actually wrong with my code and inspired me to file this issue.

**Comment by Aladdin-ADD** [comment]
this was happening when disabling a non-existent rule. https://github.com/eslint/eslint/issues/9505

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get consensus from the team, so I'm closing it. We define [consensus](http://eslint.org/docs/maintainer-guide/issues.html#consensus) as having three :+1:s from team members, as well as a team member willing to champion the proposal. This is a high bar by design -- we can't realistically accept and maintain every feature request in the long term, so we only accept feature requests which are useful enough that there is consensus among the team that they're worth adding.
